.github/labels.yml

```diff
@@ -0,0 +1,49 @@
+# Main categories
+- name: 'breaking'
+  color: 'b60205'
+  description: 'Breaking API change or removal of functionality'
+- name: 'bug'
+  color: 'd73a4a'
+  description: "Something isn't working"
+- name: 'documentation'
+  color: 'a2eeef'
+  description: 'Improvements or additions to documentation'
+- name: 'feature'
+  color: '0075ca'
+  description: 'Added or improved external functionality'
+
+# Guidance
+- name: 'good first issue'
+  color: '90ee90'
+  description: 'Good for newcomers'
+- name: 'help wanted'
+  color: '008672'
+  description: 'Please help if you can'
+- name: 'question'
+  color: 'fbca04'
+  description: 'Further information is needed'
+
+# Refactoring
+- name: 'dependencies'
+  color: 'edc287'
+  description: 'Change to dependencies'
+- name: 'refactoring'
+  color: 'edc287'
+  description: 'Improvement to code without change in functionality'
+- name: 'utility'
+  color: '1d76db'
+  description: 'Changed internal functionality'
+- name: 'tests'
+  color: 'edc287'
+  description: 'Tests'
```

Comment:
🤔 Hmm..... I'm wondering if it would be good to maybe change it to convey something like "This issue only introduces, refactors, or changes test code"

Reply:
@amostergaard Sure! Except I'd tweak it and say that it "creates, modifies, or removes" tests (since changes in functionality and refactoring both are modifications to the code). Or "creates, refactors, modifies functionality, or removes"? (But then it's getting wordy.)

Reply:
I like "creates, modifies, or removes". Would it be okay if I adopted that?